### PR TITLE
8627 debtor table fix

### PR DESCRIPTION
**Pull request by jordiwes**
*Issue #:* /bcgov/entity#8627

*Description of changes:*
- fix debtor table alignment

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/components/parties/Debtors.vue [hunk 1]
```diff
@@ -70,14 +70,18 @@
               :key="row.item.id"
               class="debtor-row"
             >
-              <td class="list-item__title title-text">
+              <td class="list-item__title title-text pl-7">
```
Comment (kialj876) [inline ppr-ui/src/components/parties/Debtors.vue]: please change this from pl-7 (28 px) to padding-left 30px so it matches the spec for the other summaries

File: ppr-ui/src/components/parties/Debtors.vue [hunk 2]
```diff
@@ -70,14 +70,18 @@
...
                 <v-row no-gutters>
-                  <v-col cols="12" style="padding-left: 14px;">
-                    <div class="icon-div mt-n1 pr-4">
+                  <v-col cols="3">
+                <div class="icon-div mt-n1 pr-4">
```
Comment (kialj876) [inline ppr-ui/src/components/parties/Debtors.vue]: indent within the v-col here and below please 🙏

**Review by kialj876 — CHANGES_REQUESTED** [review]
Hey Steph this is a very nice change, but can you please add it to the secured party, registering party and base summary components so they are consistent? We'll have to consolidate all these at some point at least for debtor/secured party (they are all 90% the same)

File: ppr-ui/src/components/parties/Debtors.vue (hunk 1 above)
Comment (jordiwes) [reply ppr-ui/src/components/parties/Debtors.vue]: done

File: ppr-ui/src/components/parties/Debtors.vue (hunk 2 above)
Comment (jordiwes) [reply ppr-ui/src/components/parties/Debtors.vue]: done

**Comment by jordiwes** [comment]
agreed, I did make them consistent for now

**Review by doug-lovett — APPROVED** [review]

**Comment by kialj876** [comment]
hey steph there is one more place: parties/summaries/base-party-summary (lots of duplicate code in there lol -- I might update the base-paty-summary soon so it can be plugged in to those other places)

**Review by kialj876 — APPROVED** [review]
thanks!

**Pull request merged**